src/object/class_object.h

```diff
@@ -773,6 +773,7 @@ struct sm_class
   unsigned has_active_triggers:1;	/* set if trigger processing is required */
   unsigned dont_decache_constraints_or_flush:1;	/* prevent decaching class constraint and flushing. */
   unsigned recache_constraints:1;	/* class constraints need recache. */
+  unsigned dont_load_index_from_heap:1;	/* don't load index, but create a new empty one when allocating an index. e.g. TRUNCATE */
```

Comment:
How about using load_index_from_heap instead of dont_load_index_from_heap?

Reply:
I agree that the negative name could be confusing, but `load_index_from_heap=true` sounds like it always loads index from a heap. How about "force_empty_index_when_alloc"?
Or do you have any idea?